Magefile.go

```diff
@@ -122,23 +123,23 @@ func (Frontend) Build() error {
 	if err != nil {
 		return err
 	}
-	return runToolboxCmd("npx", "@grafana/toolkit", "plugin:build")
+	return runToolboxCmd("npx", "@grafana/toolkit@"+grafanaVersion, "plugin:build")
```

Comment:
not sure that's the best way but since toolkit for grafana 8 is out it was using it and the build was not working. ideally npx would use the package from local node_modules directory (didn't dig it yet since it was out of this feature scope)